documentation/get-started/concepts.en.md

```diff
@@ -1,46 +1,37 @@
 ---
-title: Concepts
+title: Principles
 tx_slug: documentation_get-started_concepts
 ---
 
-# Concepts
+# Principles
 
-QField was designed with a few key concepts in mind.
+QField is designed with a few key principles in mind.
 
-## Keep it simple
 
-The requirements on the field are not the same as on a desktop. The
-screen is smaller, the input devices are different and the tasks are
-different.
+## Keeping it simple
 
-QField aims to help users to perform the tasks they need to do without
-cluttering the user interface. This means, that only tasks which need to
-be done on the field are availble from the interface. Everything else is
-not.
+The requirements in the field are not the same as on a desktop.
+Firstly, when using a phone or a tablet, the canvas to work on is much more limited than a computer monitor.
+Secondly, devices for data collection as well as the individual tasks that need to be performed are vastly different.
 
-This means that everything like layer styling, form definitions and
-other project setup steps should be done on a computer with QGIS
-installed first.
+QField aims to help users do fieldwork without the nuisance of a cluttered user interface.
+In other words, only the relevant parts of a task are accessible on the interface, while less important things remain out of sight.
 
-## Be compatible with QGIS
+Before heading out to the field though, preliminary steps such as layer styling, tailored forms, or any other project configurations should first be done on the QGIS desktop version.
 
-QField is based on QGIS. It is not a rebuild of QGIS it really *does*
-use QGIS libraries. The rendering engine is exactly the same as in QGIS
-for desktop and your project will therefore look exactly the same on
-your mobile device as it does on your computer.
+## Compatibility with QGIS
 
-If something is already available as a configuration option in a QGIS
-project, it should not be re-invented. QField therefore uses the same
-edit widgets as QGIS desktop does. If a project is already configured
-for the desktop, it should just run on mobile as well.
+Since QField relies on existing QGIS Libraries, it is not an extensive rebuild of QGIS. 
+The rendering engine in QField is the same as the one used in desktop QGIS, ensuring that projects will appear virtually identical in both environments.
 
-Remember, this is just the *concept*. This is what we have in mind when
-we develop QField. It does not mean that it is already completely there
-yet.
+Configurations options prepared in QGIS beforehand don't need to be recreated, which is why QField uses the same edit widgets as QGIS desktop does. 
+As a result, projects configured on the desktop should work seamlessly on the mobile app.
+
+These principles have so far informed our development and design of QField, and will continue to do so in the future. 
+Even though QField has been around for some time now, it is still a work in progress.
```

Comment:
```suggestion
Even though QField has been around for some time now, it is still a work in progress.
```

I think this can be rephrased like:
In the last few years QField proved to be the fieldworker's best-friend and in continues to evolve with new features handling even broader range of needs.

Reply:
Sounds great. Upon rereading my sentence, I think I may have understated the power of QField, so totally agree with you on that one.

Reply:
Added some minor stylistic/grammatical changes:

```In the last few years, QField proved to be the fieldworker's best friend and continues to evolve with new features handling an even broader range of needs.
```